common/lib/capa/capa/capa_problem.py

```diff
@@ -814,6 +814,10 @@ def _extract_html(self, problemtree):  # private
 
         if tree.tag in html_transforms:
             tree.tag = html_transforms[problemtree.tag]['tag']
+            if problemtree.tag == 'problem':
+                tree.set('class', 'questions')
```

Comment:
I think this might be better if the class name was "problem" so that it maps to the olx a little better.